test/hydro/hagen_poiseuille_equation_test.py

```diff
@@ -0,0 +1,52 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import Quantity, units, errors, convert_to
+from symplyphysics.laws.hydro import hagen_poiseuille_equation
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    mu = Quantity(0.000894 * units.pascal * units.second)
+    l = Quantity(0.1 * units.meter)
+    q = Quantity(1 * (units.meter**3 / units.second))
+    r = Quantity(0.1 * units.meter)
+    Args = namedtuple("Args", ["mu", "l", "q", "r"])
+    return Args(mu=mu, l=l, q=q, r=r)
+
+
+def test_basic(test_args):
+    result = hagen_poiseuille_equation.calculate_delta_p(test_args.mu, test_args.l, test_args.q, test_args.r)
+    result_pressure = convert_to(result, units.pascal)
+    assert result_pressure == approx(2.276, 0.001)
```

Comment:
It is advised to use some data from the external sources to verify the law. Only use synthetic data if there are no other examples, or if calculations are trivial.

Reply:
Isn't it trival calculation?

Reply:
Might be easier. This time I approve it, but next time check for external sources first.